### DISCUSSION TITLE
关于 MCIM 的一些事情

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#4867] [讨论]
最近加入 PCL2 的 MCIM 模组镜像，相信大家都知道

MCIM 的文件分发那一部分是 OpenMCIM，我开发的一个项目 [Open93AtHome-V3](https://github.com/SaltWood-Studio/Open93AtHome-V3) ，本来是想实现一个兼容 OpenBMCLAPI 节点的第三方主控，没人用

但 z0z0r4 在大概9月7日找到我，询问我关于将 Open93AtHome 应用到 MCIM 上的可能性，于是我把插件系统做出来了，并且也写了一个插件实现了 MCIM 文件列表的拉取

但是碍于当时已经开学，我和他的沟通不及时，导致这东西大概约三周之后才勉强能用。现在经过一段时间的磨合和整理，这东西也是投入生产了，并且也成功地稳定地运行了一周时间没崩过

但是呢……怎么说呢，我毕竟也是上高一了，还是住宿，z0z0r4 也是高三生，就我一个人维护 Open93AtHome 确实挺困难的，比如说插件文件不更新的问题，我当时在学校推逻辑的时候发现了这个问题，但是直到我周末回到家才得以解决。

而且现在正在国庆假期期间，压力明显增大，虽然说没被打爆，但是由于我推送代码的习惯，确实也造成了一些生产事故（因此学会了先推到 dev 分支进行测试）

因此，我想我能不能找到一些社区维护者，协助我维护 Open93AtHome，以保证 MCIM 服务较为稳定……

**Comment by MoYuan-CN** [comment]
想知道一下技术栈是啥，如果匹配我会的技术栈的话可以有空的时候来帮忙

**Reply by WorldHim** [reply]
ts 为主

**Reply by SALTWOOD** [reply]
TypeScript 为主，面板是 Vuetify + JS
不过和其他语言感觉语法上没差很多（？
